### DISCUSSION TITLE
What is the release policy?

### DISCUSSION THREAD

**Discussion by jdblischak** [conda-forge/miniforge#329] [Q&A] [answered]
I noticed that the last few [releases](https://github.com/conda-forge/miniforge/releases) have been marked "pre-release", and the latest official release remains [4.12.0-2](https://github.com/conda-forge/miniforge/releases/tag/4.12.0-2) from May. From the [Release instructions](https://github.com/conda-forge/miniforge#release), it is unclear to me what the decision process is. Is the delay to be consistent with the Miniconda distributions? There doesn't appear to be a Miniconda 4.13.0 [hashes](https://docs.conda.io/en/latest/miniconda_hashes.html) despite [conda 4.13.0](https://github.com/conda/conda/releases/tag/4.13.0) being available since May.

And I guess my ultimate question is, should I delay upgrading to Miniforge/Mambaforge 4.13.0? I maintain a shared Mambaforge installation for multiple HPC users, and I don't want to upgrade if 4.13.0 isn't deemed production ready.

PS Thanks so much for creating and maintaining these distributions. They make it so much easier to install conda/mamba!

**Comment by jdblischak** [comment]
Actually, I just realized that Mambaforge isn't included the latest 4.13.0 release. Was there an issue with the CI?

**Reply by hmaarrfk** [reply]
Yeah. It seems something happened. Which is why the pretelease didn't get converted.

**Comment by hmaarrfk** [comment, marked as answer]
Honestly, conda-forge moves so fast that releasing is somewhat moot in my opinion. I know it is annoying, but i would almost always suggest to download, install, update. 

Other maintainers had asked to hold off a critical update, but i feel like we can release it now. That should fix release process. But i want to give them time to respond to my question

**Comment by hmaarrfk** [comment]
I might end up closing discussion and just having issues be discussions. Otherwise, it gets too hard to manage. But thank you for asking the question.

Even in my "releases don't matter" opinion, i think we are due for one.

**Comment by jdblischak** [comment]
Ok, thanks for the quick clarification. My main concern had been that the label "pre-release" suggested they were not fully ready for wide-spread usage (ie early adopters only)

**Reply by hmaarrfk** [reply]
it is mostly to ensure that the "latest" link is always valid for all the distributions we support:

see: https://github.com/conda-forge/miniforge#release=

**Comment by ngam** [comment]
I believe the problem is boa, which is used in the CI...

https://github.com/conda-forge/boa-feedstock/blob/b7586b08a14b1400e4104f686925869bd3b99ac1/recipe/meta.yaml#L30

**Comment by jdblischak** [comment]
@hmaarrfk Thanks for updating and making a new release. I successfully installed Mambaforge [4.13.0-1](https://github.com/conda-forge/miniforge/releases/tag/4.13.0-1)
